README.md

```diff
@@ -47,7 +47,7 @@ For these applications, LangChain simplifies the entire application lifecycle:
 - **`langchain-community`**: Third party integrations.
   - Some integrations have been further split into **partner packages** that only rely on **`langchain-core`**. Examples include **`langchain_openai`** and **`langchain_anthropic`**.
 - **`langchain`**: Chains, agents, and retrieval strategies that make up an application's cognitive architecture.
-- **`[LangGraph](https://python.langchain.com/docs/langgraph)`**: A library for building robust and stateful multi-actor applications with LLMs by modeling steps as edges and nodes in a graph.
+- **[`LangGraph`](https://python.langchain.com/docs/langgraph)**: A library for building robust and stateful multi-actor applications with LLMs by modeling steps as edges and nodes in a graph.
```

Comment:
This

> [**`LangGraph`**](https://python.langchain.com/docs/langgraph) 

would be more precise.

Reply:
Sorry, I can't figure out what your suggestion is, could you make it via the suggestion tool in GitHub comments? Your link renders similar to this markdown, so am not sure the diff.

Reply:
If you chose "Quote reply" on my message, you'll see the proposed formatting.

Reply:
```suggestion
- [**`LangGraph`**](https://python.langchain.com/docs/langgraph): A library for building robust and stateful multi-actor applications with LLMs by modeling steps as edges and nodes in a graph.
```

Reply:
Ah thanks! Makes sense but the links later use bold outside so will let reviewers decide. I just don't want it to be broken, so not fussed either way.